app/controllers/checkout_controller.rb

```diff
@@ -81,17 +81,19 @@ def ensure_order_not_completed
   def load_order
     @order = current_order
 
-    redirect_to(main_app.shop_path) && return if redirect_to_shop?
+    if order_invalid_for_checkout?
+      Bugsnag.notify("Notice: invalid order loaded during Stripe processing", order: @order) if valid_payment_intent_provided?
```

Comment:
I'm not yet familiar with the current bugsnag config but I suppose some context is recorded with each notification like when using Sentry.
Would it possible to add the reason of why the order is invalid in the notification?

Reply:
The order record is given in the context so that we can see what may have gone wrong.

Reply:
@mkllnk Yes sorry for not getting back to you faster... It's good that the record is recorded in the context, but if the function `order_invalid_for_checkout?` actually looks for an error, it might be great to add that error to the bugsnag context, it would faster than looking at the code and checking manually why the order is not valid.